src/library_signals.js

```diff
@@ -116,34 +64,14 @@ var funs = {
       if (__sigalrm_handler) {{{ makeDynCall('vi', '__sigalrm_handler') }}}(0);
     }, seconds*1000);
   },
-  ualarm: function() {
-    throw 'ualarm() is not implemented yet';
-  },
-  setitimer: function() {
-    throw 'setitimer() is not implemented yet';
-  },
-  getitimer: function() {
-    throw 'getitimer() is not implemented yet';
-  },
-
-  pause__deps: ['$setErrNo', '$ERRNO_CODES'],
-  pause: function() {
```

Comment:
where is `pause` added from musl?

Reply:
Ditto, we were already building `unistd/pause.c`.